onnx_chainer/export.py

```diff
@@ -203,6 +196,10 @@ def export(model, args, filename=None, export_params=True,
         train (bool): If True, output computational graph with train mode.
         return_flat_inout (bool): If set True, return ONNX model with flat
```

Comment:
Sorry I haven't thought of this before, but how about saying this argument is mainly for export_testcase and may be changed in future? I think we don't want users to start using this now yet. Please check the next comment, too.

Reply:
Oh, the "next" was actually the previous one on github UI. I meant my comment for `external_converters (dict): ...`